### DISCUSSION TITLE
Script only deletes 25 messages

### DISCUSSION THREAD

**Discussion by pansear** [victornpb/undiscord#302] [1. Questions & Answers]
I've been getting this error for a few days now
also, I keep seeing empty pages whenever I look up messages, is it just discord being weird? i don't know
![image](https://user-images.githubusercontent.com/85607515/148316657-655a0aeb-7cdd-4a0f-88f2-09c10871e2e4.png)
![image](https://user-images.githubusercontent.com/85607515/148316808-8232caaa-af87-4599-adb3-c80de3bd81eb.png)

**Comment by gqqjhffr** [comment]
I'm also getting this.
